express/src/controllers/clients.ts

```diff
@@ -206,11 +219,16 @@ export const processChangeRedirectUrlsForm: RequestHandler = async (req, res) =>
 };
 
 export const showProcessChangeUserAttributesForm: RequestHandler = async (req, res) => {
-    const userAttributes = nonNull(req.query.userAttributes?.toString()).split(" ");
-    const email: boolean = userAttributes.includes("email");
-    const phone: boolean = userAttributes.includes("phone");
-    const offline_access: boolean = userAttributes.includes("offline_access");
-
+    let email = false;
+    let phone = false;
+    let offline_access = false;
+
+    if (req.query.userAttributes) {
+        const userAttributes = req.query.userAttributes?.toString().split(" ");
+        email = userAttributes.includes("email");
+        phone = userAttributes.includes("phone");
+        offline_access = userAttributes.includes("offline_access");
+    }
```

Comment:
The user attributes are checkboxes, the controller should return the names of the checked boxes. The govuk checkboxes components handles this.